packages/libtasn1/build.sh

```diff
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://www.gnu.org/software/libtasn1/
 TERMUX_PKG_DESCRIPTION="This is GNU Libtasn1, a small ASN.1 library"
 TERMUX_PKG_LICENSE="LGPL-2.1"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=4.16.0
+TERMUX_PKG_VERSION=4.17.0
 TERMUX_PKG_SRCURL=https://gitlab.com/gnutls/libtasn1.git
-TERMUX_PKG_GIT_BRANCH=libtasn1_${TERMUX_PKG_VERSION//./_}
+TERMUX_PKG_GIT_BRANCH=v${TERMUX_PKG_VERSION}
```

Comment:
You can remove this line completely, `v$TERMUX_PKG_VERSION` is the default branch used: https://github.com/termux/termux-packages/blob/ebdcae3eac76314486613ad3704574063652c242/scripts/build/get_source/termux_git_clone_src.sh#L9